.ci/integration_test_without_k8s

```diff
@@ -0,0 +1,189 @@
+    rm -f ./etcd-${ETCD_VER}-linux-amd64
+    rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+    echo "Successfully installed etcd."
+
+    echo "Installing etcdbrctl."
+    LOCAL_BUILD=1 .ci/build
+    chmod +x bin/etcdbrctl
+    mv ./bin/etcdbrctl /usr/local/bin/etcdbrctl
+    echo "Successfully installed etcdbrctl."
+
+    echo "Installing awscli"
+    apt update && apt install python-pip
+    pip install awscli
+    echo "Successfully installed awscli"
+}
+
+function get_commit_id() {
+  git_branch=`cat .git/HEAD| cut -f 2 -d ' '`
+  NAMESPACE=commit`cat .git/$git_branch | cut -f 1 -d ' '`
+}
+
+#############################
+#        AWS Setup          #
+#############################
+
+function create_aws_secret() {
+  echo "Fetching aws credentials from secret server..."
+  ACCESS_KEY_ID=`/cc/utils/cli.py config attribute --cfg-type aws --cfg-name etcd-backup-restore --key aws_access_key_id`
+  SECRET_ACCESS_KEY=`/cc/utils/cli.py config attribute --cfg-type aws --cfg-name etcd-backup-restore --key aws_secret_access_key`
+  REGION=`/cc/utils/cli.py config attribute --cfg-type aws --cfg-name etcd-backup-restore --key region`
+  echo "Successfully fetched aws credentials from secret server."
+  
+  echo "Creating aws credentials for API access."
+  mkdir $HOME/.aws
+  cat << EOF > $HOME/.aws/credentials
+[default]
+aws_access_key_id = $ACCESS_KEY_ID
+aws_secret_access_key = $SECRET_ACCESS_KEY
+EOF
+cat << EOF > $HOME/.aws/config
+[default]
+region = $REGION
+EOF
+  echo "Successfully created aws credentials."
+}
+
+function delete_aws_secret() {
+  rm -rf $HOME/.aws
+}
+
+function create_etcd_data_directory() {
+  #ETCD_DATA_DIR='/tmp/$NAMESPACE/etcd-data'
+  # For local testng
+  ETCD_DATA_DIR='$PWD/etcd-data'
+  mkdir -p $ETCD_DATA_DIR
+}
+
+function create_s3_bucket(){
+    aws s3api create-bucket --bucket $NAMESPACE --region $LOCATION --create-bucket-configuration LocationConstraint=$LOCATION
+}
+
+function setup-aws-infrastructure() {
+  create_aws_secret
+  create_s3_bucket
+  create_etcd_data_directory
+}
+
+#############################
+#        Azure Setup        #
+#############################
+function create_azure_secret() {
+  echo "Creating Azure secret"  
+}
+
+#############################
+#        GCP Setup          #
+#############################
+function create_gcp_secret() {
+echo "Creating GCP secret" 
+}
+
+#############################
+#        Openstack Setup    #
+#############################
+function create_openstack_secret() {
+echo "Creating Openstack secret" 
+}
+
+##############################################################################
+function setup_test_cluster() {
+  # create namespace unique to commit
+  echo "Creating namespace specific for current test..."
+  get_commit_id
+  # setup secrets
+  setup-aws-infrastructure
+  create_gcp_secret
+  create_azure_secret
+  create_openstack_secret
+  #test_aws_integration
+
+}
+
+function delete_test_cluster(){
```

Comment:
Rename it to `cleanup_test_environment`. Put a code to remove etcd data directory, bucket. unset environment variables etc. i.e all code related to cleanup.